src/Table.luna

```diff
@@ -152,21 +395,78 @@ class Table:
     def sort sortBy:
         sortByColumns = sortBy.each (colName, order, pos): (self.column colName, order, pos)
         self.sortByColumn sortByColumns
+
+    # Remove rows with missing values.
+    #
+    # > import Dataframes.Table
+    # > import Dataframes.IO 
+    # > table = CSVParser.readFile "./data/simple_empty.csv"
+    # > newTable = table.dropNa
+    #
+    # `return`: `Table` value without rows containing NA records.
```

Comment:
We need to unify terms:
* NA
* missing value
* null
* NAN
* hole

The Arrow's term is "null value". Certainly better than "nan" that makes sense only for numeric columns.